src/polyfills/details/_base.scss

```diff
@@ -6,6 +6,10 @@
 summary {
 	// Make sure summary remains visible
 	display: list-item !important;
+	list: {
+		style: none;
```

Comment:
Got it, but I was confuse seeing two consecutive "style" and seems to me generic at my first look.

As I suggested in my https://github.com/wet-boew/wet-boew/pull/7822#discussion_r97809071 may be to add a comments before explaining that we need to declare in CSS two  list-style for browser that do not recognize ```disclosure-closed``` will add more clarity